dev/benchmarks/macrobenchmarks/lib/src/web/bench_build_image.dart

```diff
@@ -61,22 +61,16 @@ class _RotatingWidget extends StatefulWidget {
 }
 
 class _RotatingWidgetState extends State<_RotatingWidget> with SingleTickerProviderStateMixin {
-  AnimationController controller;
-
-  @override
-  void initState() {
-    super.initState();
-    controller = AnimationController(
+  late final AnimationController controller = AnimationController(
       duration: const Duration(milliseconds: 200),
       vsync: this,
     )..repeat();
```

Comment:
For all of these: the initialization can be set in the `late final`, but keep the methods that act on it in `initState`, so move the `repeat()` part back into the `initState`

Reply:
Sure thanks